src/stable/lib/did_file/visitor/quote_candid_name.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
Normalize name and JSDocs to use the term `escaping`.

Reply:
done